spec/base_spec.rb

```diff
@@ -0,0 +1,80 @@
+require_relative '../lib/base'
+
+RSpec.describe Ipfs::Base16 do
+  #let(:invalid_byte_set) { 'fezhgzhze6351498' }
+  #let(:valid_byte_set) { 'cbdafe651498465bcdeaf' }
+  #let(:invalid_byte_set) { 'uluiil1658gfh' }
+  #let(:valid_byte_set) { 'f5c1d23ea6e9' }
```

Comment:
comments in commit

Reply:
Sorry about that, forgot to take them off